target/linux/ath79/dts/qca9558_jjplus_jwap230.dts

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
```

Comment:
dts-v1 can be dropped and qca955x.dtsi should be moved before the other includes.

Reply:
The file was updated accordingly.